### ISSUE TITLE
Bluetooth Issues

### ISSUE THREAD

**Issue by QuiteAFancyEmerald** [gordonbay/Windows-On-Reins#12]
Are the issues with Bluetooth to be expected? Ran the script in a controlled environment but can't seem to fix anything.

I've tried reinstalling drivers, etc. to no avail. Any solutions to this?

**Comment by gordonbay** [comment]
Yes, there are several issues with bluetooth. Considering that i don`t use bluetooth, i don`t have any motivation to track what is messing it up and create a beBluetoothSafe config. At least not right now.

**Comment by QuiteAFancyEmerald** [comment]
Oh alright no worries
